### PR TITLE
Added Redux DevTools.

### LINKED ISSUE #52: Add Redux DevTools
https://github.com/gaearon/redux-devtools

### PR DESCRIPTION
**Pull request by Nitive**
This close #52

### CODE REVIEW

**Comment by Nitive** [comment]
screen shot 2015-09-17 at 01 42 23

**Comment by dimaip** [comment]
Looks fine :+1:

**Comment by SeqviriouM** [comment]
++

File: app/app.jsx [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
-store.dispatch(addMessage(Map({id: 0, channelId: 0, senderId: 0, text: 'test message'})));
-store.dispatch(addMessage(Map({id: 1, channelId: 0, senderId: 1, text: 'test message1'})));
-store.dispatch(addMessage(Map({id: 2, channelId: 0, senderId: 0, text: 'test message2'})));
-store.dispatch(addMessage(Map({id: 3, channelId: 0, senderId: 1, text: 'test message3'})));
-
-const msgs = store.getState().messages;
 
 startSocketClient();
 
+@connect(state => ({ messages: state.messages }))
```
Comment (dimaip) [inline app/app.jsx]: Btw, how does this work without passing component to a second function of the decorator?

Comment (dimaip) [inline app/app.jsx]: Nevermind, I think I get it.

**Pull request merged**